src/radio/styled-components.js

```diff
@@ -73,31 +72,35 @@ export const RadioMark = styled('span', props => {
             ? colors.mono400
             : null,
   };
+
   return {
-    backgroundColor: 'transparent',
+    backgroundColor: 'white',
     borderRadius: '50%',
-    borderWidth: $checked ? scale300 : scale0,
-    width: $checked ? scale100 : scale600,
-    height: $checked ? scale100 : scale600,
-    backgroundImage: null,
-    flex: '0 0 auto',
+    height: $checked ? sizing.scale100 : sizing.scale600,
     transitionDuration: animation.timing100,
     transitionTimingFunction: animation.easeOutCurve,
-    borderStyle: 'solid',
-    borderColor: getBorderColor(props),
-    display: 'inline-block',
-    verticalAlign: 'middle',
-    backgroundRepeat: 'no-repeat',
-    backgroundPosition: 'center',
-    marginTop: $theme.sizing.scale200,
-    marginBottom: $theme.sizing.scale200,
-    marginLeft: $theme.sizing.scale200,
-    marginRight: $theme.sizing.scale200,
+    width: $checked ? sizing.scale100 : sizing.scale600,
     ':hover': activeStyle,
     ':active': activeStyle,
   };
 });
 
+export const RadioMarkOuter = styled('div', props => {
+  const {sizing} = props.$theme;
+
+  return {
+    alignItems: 'center',
+    backgroundColor: getBorderColor(props),
+    borderRadius: '50%',
+    display: 'flex',
+    height: sizing.scale700,
+    justifyContent: 'center',
+    margin: sizing.scale200,
```

Comment:
use longhand style props instead:
```
    marginTop: $theme.sizing.scale200,	
    marginBottom: $theme.sizing.scale200,	
    marginLeft: $theme.sizing.scale200,	
    marginRight: $theme.sizing.scale200,
```

Reply:
just for me information, can you help me why we use longhands styles?

Reply:
ah, right. thanks!

Reply:
> just for my information, can you help me why we use longhands styles?

We use longhand styles because mixing shorthand and longhand styles can cause issues (https://github.com/styletron/styletron/issues/68 for example) and standardizing on longhand styles allows us to be more granular with overrides and not have to recreate the full shorthand style that's being overridden.

Reply:
makes sense, thanks Mark!